v0/destinations/tiktok_ads/transform.js

```diff
@@ -110,6 +204,7 @@ const trackResponseBuilder = async (message, { Config }) => {
   response.method = defaultPostRequestConfig.requestMethod;
   response.endpoint = TRACK_ENDPOINT;
   response.body.JSON = removeUndefinedAndNullValues(payload);
+
   return response;
```

Comment:
Line no 143 to 208 is redundant. We can remove that by calling `getTrackResponse` function.